add-column.md

```diff
@@ -18,6 +18,121 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+~~~
+
+
+### Add Multiple Columns
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN location STRING, ADD COLUMN amount DECIMAL;
+~~~
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------+-----------+
+|   Field   |       Type        | Null  | Default |  Indices  |
++-----------+-------------------+-------+---------+-----------+
+| id        | INT               | false | NULL    | {primary} |
+| balance   | DECIMAL           | true  | NULL    | {}        |
+| names     | STRING            | true  | NULL    | {}        |
+| location  | STRING            | true  | NULL    | {}        |
+| amount    | DECIMAL           | true  | NULL    | {}        |
++-----------+-------------------+-------+---------+-----------+
+
+~~~
+
+### Add a Non-Null Column with a Default Value
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN interest DECIMAL NOT NULL DEFAULT (DECIMAL '1.3');
+~~~ 
+
+~~~ sql
+> SHOW COLUMNS FROM accounts;
+~~~ 
+
+~~~
++-----------+-------------------+-------+---------------------------+-----------+
+|   Field   |       Type        | Null  |          Default          |  Indices  |
++-----------+-------------------+-------+---------------------------+-----------+
+| id        | INT               | false | NULL                      | {primary} |
+| balance   | DECIMAL           | true  | NULL                      | {}        |
+| names     | STRING            | true  | NULL                      | {}        |
+| location  | STRING            | true  | NULL                      | {}        |
+| amount    | DECIMAL           | true  | NULL                      | {}        |
+| interest  | DECIMAL           | false | ('1.3':::STRING::DECIMAL) | {}        |
++-----------+-------------------+-------+---------------------------+-----------+
+~~~
+
+
+### Add a Non-Null Column with Unique Values
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN cust_number DECIMAL UNIQUE NOT NULL;
+~~~ 
+ 
+
+### Add a Column with Collation
+
+~~~ sql
+> ALTER TABLE accounts ADD COLUMN more_names STRING COLLATE en;
+~~~ 
+ 
+
+### Add a Column to a Column Family
+
+~~~ sql
+# Add a Column to a Column Family
+> ALTER TABLE accounts ADD COLUMN location1 STRING CREATE FAMILY new_family;
+~~~ 
+
+
+~~~ sql
+# Add a Column to an Existing Column Family
+> ALTER TABLE accounts ADD COLUMN location2 STRING FAMILY existing_family;
+~~~
+
+### Add a Column Using CREATE IF NOT EXISTS FAMILY
```

Comment:
This is what I have now:

### Add a column to a column family

# Add a Column to a New Column Family
>ALTER TABLE accounts ADD COLUMN location1 STRING CREATE FAMILY new_family;

# Add a Column to an Existing Column Family
> ALTER TABLE accounts ADD COLUMN location2 STRING FAMILY existing_family;

#Add a Column Using `CREATE IF NOT EXISTS FAMILY`
> ALTER TABLE accounts ADD COLUMN new_name STRING CREATE IF NOT EXISTS FAMILY f1;

Does this look okay?